src/pages/ComponentsLibrary/ComponentsLibrary.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { Loader } from '../../components/Loader';
```

Comment:
❗️
Очень хочется алиасы, чтобы было так
```suggestion
import { Loader } from 'components/Loader';
```

Reply:
Добавим, но чуть позже, в другом ПР)